CppCoreGuidelines.md

```diff
@@ -14196,6 +14198,30 @@ Really, "all bets are off" if you get a data race.
 Actually, it can be worse still: by looking at the generated code you might be able to determine where the stray jump will go for a given value;
 this can be a security risk.
 
+##### Example, better
```

Comment:
I think what the rule needs is an 'Example, Good'.  Currently it has three bad examples and no solutions.

Not sure what is best but maybe combine the first and third into one bad example and drop the second?  Then show how to fix it in a good example?  Somehow writing good bad examples seems to be harder than writing good good examples.

Reply:
> I think what the rule needs is an 'Example, Good'. Currently it has three bad examples and no solutions.
> 
> Not sure what is best but maybe combine the first and third into one bad example and drop the second? Then show how to fix it in a good example? Somehow writing good bad examples seems to be harder than writing good good examples.

A 'good'-marked example would be very nice to study and learn from.